launcher/Unmanic Launcher.au3

```diff
@@ -30,10 +36,11 @@ Func Tray_Menu()
         TraySetState($TRAY_ICONSTATE_SHOW) ; Show the tray menu.
 
         While 1
+				Sleep(100)
                 Switch TrayGetMsg()
                         Case $idAbout ; Display a message box about the AutoIt version and installation path of the AutoIt executable.
                                 MsgBox($MB_SYSTEMMODAL, "", "Unmanic Launcher" & @CRLF & @CRLF & _
-                                                "Version: 0.1" & @CRLF)
+                                                "Version: 0.2.1" & @CRLF)

```

Comment:
Is it possible to read the version from a config file like the one in the root? Or possibly an environment variable?

Reply:
Not wanting this fix now... Just consideration for the future